utils/progressbar/transferfileprogressbar.go

```diff
@@ -0,0 +1,147 @@
+package progressbar
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/vbauerster/mpb/v7"
+	"sync/atomic"
+	"time"
+)
+
+// TransferProgressMng Managing all self-hosted to SaaS transfer progress details.
```

Comment:
TransferProgressMng Managing all self-hosted to SaaS transfer progress details.
-->
TransferProgressMng provides progress indication for the `jf rt transfer-files` command.